microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1184,46 +1184,73 @@ Get information about running WebView2 processes, exiting processes, and failed
 
 * `CoreWebView2` Class:
    * [CoreWebView2.BrowserProcessId Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.browserprocessid)
+   * [CoreWebView2.Reload Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.reload)
    * [CoreWebView2.ProcessFailed Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.processfailed)
 * [CoreWebView2BrowserProcessExitedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2browserprocessexitedeventargs)
+* `CoreWebView2Controller` Class:
+   * [CoreWebView2Controller.Close Method](/dotnet/api/microsoft.web.webview2.core.corewebview2controller.close)
 * `CoreWebView2Environment` Class:
-   * [CoreWebView2Environment.BrowserProcessExited Event](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.browserprocessexited)
+   * [CoreWebView2Environment.FailureReportFolderPath Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.failurereportfolderpath)
```

Comment:
@mikehoffms `Reload`, `Close` and `FailureReportFolderPath` are only related to process management in that they can be used when handling process-related events. I think it could be confusing listing them in the same place as the other APIs which are directly related to process management. Can we move them into a separate list?

Reply:
so far, in local file, i added todo comments on those 3 items:
<!--todo: move to section other than Process Management (x3) -->

Reply:
Covered by plan in other [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2670/files#r1258907566).  Possibly close this thread.